documentation/src/main/resources/_posts/2023-08-23-ditto-benchmark.md

```diff
@@ -0,0 +1,465 @@
+---
+title: "Eclipse Ditto Benchmark"
+published: false
+permalink: 2023-08-23-ditto-benchmark.html
+layout: post
+author: vvasilevbosch
+tags: [blog]
+hide_sidebar: true
+sidebar: false
+toc: true
+
+---
+
+This blog post is presenting a benchmark of Eclipse Ditto. It consists of a few scenarios to cover most important ditto functionalities, test the performance and provide a tuning guide. This benchmark is done via the [benchmark-tool](https://github.com/bosch-io/ditto/tree/master/benchmark-tool), based on [k6](https://k6.io/) load testing tool
+
+## Setup and used tools
+
+- EKS cluster using [m5.4xlarge](https://aws.amazon.com/ec2/instance-types/m5/) nodes.
+
+- Deployed Ditto using [ditto's official helm chart](https://artifacthub.io/packages/helm/eclipse-iot/ditto).
```

Comment:
Ditto's official Helm chart is no longer at that location.
See: https://github.com/eclipse-ditto/ditto/tree/master/deployment/helm

Did you run the benchmarks with the old chart or with the new one?

Reply:
Chart version for the benchmarks is 3.0.0. I guess that's the old one, should I keep the old link or update with the new one?

Reply:
I would mention that you did the benchmark with 3.0.0, however reference to the up-to-date chart and not the deprecated one.